### DISCUSSION TITLE
Are circular progress bars possible?

### DISCUSSION THREAD

**Discussion by ghost** [beeware/toga#2879] [Q&A]
I've found some info to do it in other gui toolkits, but nothing solid for Toga. I don't see too much in the Toga docs that talks about customizing the ProgressBar to this extent, so I don't have high hopes.

Just curious if anyone knows whether it's possible at all or can point me in the right direction on how to do it if it can.

**Comment by freakboy3742** [comment]
Depends what you mean by "circular progress bar". If you mean a "spinner", then [ActivityIndicator](https://toga.readthedocs.io/en/stable/reference/api/widgets/activityindicator.html) is the widget you want.
